### ISSUE TITLE
does not work on samsung devices

### ISSUE THREAD

**Issue by namikkaya** [fishwjy/VideoCompressor#38]
creates a blank video file very quickly. Tried on Samsung device does not work.

androidone (brand: general mobile) works on the device.

**Comment by richanshah** [comment]
same issue! did you found any solution for it?

**Comment by namikkaya** [comment]
> same issue! did you found any solution for it?

Yes. I'm not using this library. ffmpeg mobile was the best solution for me

**Comment by Gurvinder963** [comment]
@namikkaya give me link of ffmpeg where video quality is good ,nd not takes much time

**Comment by prashantd123** [comment]
give me link of  ffmpeg. anybody please...

**Comment by prashantd123** [comment]
> > same issue! did you found any solution for it?
> 
> Yes. I'm not using this library. ffmpeg mobile was the best solution for me

give me link of ffmpeg mobile. anybody please...

**Comment by xiaobaDev** [comment]
> > > same issue! did you found any solution for it?
> > 
> > 
> > Yes. I'm not using this library. ffmpeg mobile was the best solution for me
> 
> give me link of ffmpeg mobile. anybody please...

https://github.com/tangpeng/FFmpegDemo
